inference-engine/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -26,6 +26,11 @@
 namespace ov {
 namespace test {
 
+std::ostream& operator <<(std::ostream& os, const InputShape& inputShape) {
+    os << inputShape.first << "_" << CommonTestUtils::vec2str(inputShape.second);
```

Comment:
I think it's incorrect. Because `InputShape` is a pair when second part is vector of target shapes. Why do you use only target shapes? Where is dynamic shape?

Reply:
fixed, the commit was cherry-picked. Has been discussed with @maxnick:   https://github.com/openvinotoolkit/openvino/pull/8387#discussion_r751342702